### DISCUSSION TITLE
Remix usage

### DISCUSSION THREAD

**Discussion by redbaron76** [tw-in-js/twind#259] [Q&A]
Any hint on how to get started with Remix framework?

I've already tried many ways but none of them worked.
Thanks

**Comment by sastan** [comment]
That is on the roadmap. I haven't used remix yet. Maybe the existing integrations can help:

- https://github.com/tw-in-js/twind/tree/next/packages/sveltekit
- https://github.com/tw-in-js/twind/tree/next/packages/gatsby-plugin

**Comment by sastan** [comment]
here you go https://www.npmjs.com/package/@twind/with-remix and an example is available as well https://github.com/tw-in-js/twind/tree/next/examples/with-remix

**Reply by redbaron76** [reply]
You are the hero of the day!
Thanks man 👍
